EventFilter/RPCRawToDigi/plugins/RPCDigiMerger.cc

```diff
@@ -0,0 +1,102 @@
+#include "EventFilter/RPCRawToDigi/plugins/RPCDigiMerger.h"
+
+#include <memory>
+
+#include "DataFormats/Common/interface/Handle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/Utilities/interface/CRC16.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+#include "DataFormats/RPCDigi/interface/RPCDigiCollection.h"
+
+using namespace edm;
+using namespace std;
+
+RPCDigiMerger::RPCDigiMerger(edm::ParameterSet const & config)
+{
+    produces<RPCDigiCollection>();
+    // if (fill_counters_) {
```

Comment:
Remove commented out lines if not needed
(Otherwise add another line of comment which explains why and when they can be needed)